### PR TITLE
add dsu to plugins that use the api

### PR DESCRIPTION
**Pull request by BlueTree242**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -51,6 +51,7 @@ _**We only give support for the latest release/dev builds; please update before
 * [MCSF (My Christian Swear Filter)](https://www.spigotmc.org/resources/mcsf.54115/)
 * [MZP-VoteParty](https://www.spigotmc.org/resources/mzp-voteparty.89754/)
 * [InteractiveChat DiscordSRV Addon](https://www.spigotmc.org/resources/interactivechat-discordsrv-addon.83917/)
+* [DiscordSRVUtils Addon](https://discordsrvutils.xyz/)
```
Comment (granny) [inline docs/index.md:54]:
```suggestion
* [DiscordSRVUtils Addon](https://www.spigotmc.org/resources/discordsrvutils.85958/)
```
The website is currently down. I don't trust it to be up 24/7 so I'd appreciate if the url was replaced with the spigot resource page.

**Review by granny — CHANGES_REQUESTED** [review]
Either the plugin name needs to be changed to not include "DiscordSRV" or a disclaimer must be added stating it's not associated with DiscordSRV at all.

**Comment by BlueTree242** [comment]
> Either the plugin name needs to be changed to not include "DiscordSRV" or a disclaimer must be added stating it's not associated with DiscordSRV at all.

There is a disclaimer at the bottom of each spigot page and website

**Comment by BlueTree242** [comment]
image

**Comment by BlueTree242** [comment]
hello?

**Comment by granny** [comment]
closing since I can't push changes to repos that are behind organizations. superceded by https://github.com/DiscordSRV/Documentation/commit/926572923a400428a08e71100e1b25664667b78e

**Pull request closed without merging**